GazeInputTest/GazeInputTest.csproj

```diff
@@ -150,7 +150,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.NETCore.UniversalWindowsPlatform">
-      <Version>6.2.10</Version>
+      <Version>6.2.11</Version>
```

Comment:
There is already 6.2.12.

Reply:
I just did this yesterday! 🤦‍♂️ I'll update.